docs/sources/enterprise/vault.md

```diff
@@ -8,10 +8,9 @@ weight = 1200
 
 # Vault integration
 
-> Only available in Grafana Enterprise v7.1+.
+If you manage your secrets with [Hashicorp Vault](https://www.hashicorp.com/products/vault), you can use them for [Configuration]({{< relref "../administration/configuration.md" >}}) and [Provisioning]({{< relref "../administration/provisioning.md" >}}).
```

Comment:
```suggestion
If you manage your secrets with [Hashicorp Vault](https://www.hashicorp.com/products/vault), you can refer to those secrets for [Configuration]({{< relref "../administration/configuration.md" >}}) and [Provisioning]({{< relref "../administration/provisioning.md" >}}) instead of exposing the secrets in plain text.
```

Reply:
Is plain text the only alternative for these values? How are they exposed? Does this mean the default configuration methods are insecure/exposed?